### DISCUSSION TITLE
Best install method for Frigate with HA on Mac ?

### DISCUSSION THREAD

**Discussion by mcdent** [blakeblackshear/frigate#2844] [Ask A Question]
Hi,
I have a Mac which I would like to dedicate to Frigate (about 5 cameras) and Home Assistant. HA will be doing some temperature monitoring, so nothing else too stressful.

I'm wondering what the best install method for this is.

The Mac only has 4GB RAM (I cant expand that, its a 2013 MacBook Air with 1.7Ghz core i7 and 512GB SSD)

I've only done installs with Raspberry pi and NUC in the past, which were all HA OS.

Thanks!

**Comment by ireuben** [comment]
I use Docker to run HA and Frigate on my Mac, and it seems to work well. You can use all the normal command line tools for running it, or it has a little GUI.

**Comment by richardkuw** [comment]
I would wipe it and install Linux. I put Ubuntu 20 on a 2015 Macbook Air, the install was trivial and everything works. I actually think Debian is a better choice for HA but you would need to investigate how well it supports your hardware. Nothing wrong with Mac OS but it's BSD based - there are so many differences from Linux and you will constantly be working around them. This is also a way to get a current, supported OS since you can't run current Mac OS on your hardware.
